tests/integration/docker/cmd_docker_devfile_push_test.go

```diff
@@ -0,0 +1,81 @@
+package docker
+
+import (
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/openshift/odo/tests/helper"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("odo docker devfile push command tests", func() {
+	var context string
+	var currentWorkingDirectory string
+	var cmpName string
+
+	// TODO: all oc commands in all devfile related test should get replaced by kubectl
+	// TODO: to goal is not to use "oc"
+	dockerClient := helper.NewDockerRunner("docker")
```

Comment:
So this is just a question, was docker already installed on the CI or did you have to do something?

Reply:
Docker was already installed on travis where these tests are running. Docker isn't available on the OpenShift CI, so the tests don't run there.

Reply:
So would you know, how to check the list of tests running on the CI like
`ci/prow/v4.1-integration-e2e-benchmark`, etc.

Reply:
The integration tests that run on the OpenShift CI are defined here: https://github.com/openshift/odo/blob/master/scripts/openshiftci-presubmit-all-tests.sh